internal/dinosaur/internal/handlers/authentication.go

```diff
@@ -32,9 +31,9 @@ func NewAuthenticationBuilder(ServerConfig *server.ServerConfig, KeycloakConfig
 			KeysURL(KeycloakConfig.OSDClusterIDPRealm.JwksEndpointURI). // sso SRE realm cert URL
 			Error(fmt.Sprint(errors.ErrorUnauthenticated)).
 			Service(errors.ERROR_CODE_PREFIX).
-			Public(fmt.Sprintf("^%s/%s/?$", routes.ApiEndpoint, routes.DinosaursFleetManagementApiPrefix)).
-			Public(fmt.Sprintf("^%s/%s/%s/?$", routes.ApiEndpoint, routes.DinosaursFleetManagementApiPrefix, routes.Version)).
-			Public(fmt.Sprintf("^%s/%s/%s/openapi/?$", routes.ApiEndpoint, routes.DinosaursFleetManagementApiPrefix, routes.Version)).
-			Public(fmt.Sprintf("^%s/%s/%s/errors/?[0-9]*", routes.ApiEndpoint, routes.DinosaursFleetManagementApiPrefix, routes.Version)),
+			Public(fmt.Sprintf("^%s/?$", ServerConfig.APIBasePath)).
+			Public(fmt.Sprintf("^%s/%s/?$", ServerConfig.APIBasePath, ServerConfig.APIVersion)).
+			Public(fmt.Sprintf("^%s/%s/openapi/?$", ServerConfig.APIBasePath, ServerConfig.APIVersion)).
```

Comment:
that raises an interesting question, when we have v2 of the API, what is implementing v1 so we support v1 and v2?

Reply:
If the state is all entirely shared from the same DB with no other caching and that the reconcile loops can be run concurrently, that works.

Reply:
> If the state is all entirely shared from the same DB with no other caching and that the reconcile loops can be run concurrently, that works.

Yes, thought it has its limits when DB changes. API versioning is something I'd wish we have an Epic for us to define our versioning strategy. I am happy to create one if we do not have it already @emmanuelbernard @davidffrench wdyt?